addons/mail/models/res_users.py

```diff
@@ -127,6 +158,72 @@ def unlink(self):
         self._unsubscribe_from_non_public_channels()
         return super().unlink()
 
+    def _notify_security_setting_update(self, subject, content, mail_values=None, **kwargs):
+        """ This method is meant to be called whenever a sensitive update is done on the user's account.
+        It will send an email to the concerned user warning him about this change and making some security suggestions.
+
+        This behavior can be manually disabled by turning off 'notify_security_updates' in the context
+
+        :param str subject: The subject of the sent email (e.g: 'Security Update: Password Changed')
+        :param str content: The text to embed within the email template (e.g: 'Your password has been changed')
+        :param kwargs: 'suggest_password_reset' key:
+            Whether or not to suggest the end-user to reset
+            his password in the email sent.
+            Defaults to True. """
+
+        if not self.env.context.get('notify_security_updates', True):
```

Comment:
That doesn't seem like a good idea. If somebody calls `write` in RPC you don't want to be able to disable it with the context. If you really want to be able to disable it via context, at least pass a value that is not RPC-injectable (e.g. a python object)

Reply:
Indeed, I will try to remove the context support in favor of a local patch in tests :+1: